src/main/scala/util/AsyncQueue.scala

```diff
@@ -96,10 +98,10 @@ class AsyncQueueSink[T <: Data](gen: T, depth: Int, sync: Int) extends Module {
   // This is safe, because isolation gates brought mem low before the zeroed widx reached us
   io.deq.bits  := RegEnable(io.mem(index), valid)
 
-  val valid_reg = AsyncResetReg(valid.asUInt)(0)
+  val valid_reg = AsyncResetReg(valid.asUInt, "valid_reg")(0)
```

Comment:
You could just use the Bool form of this like you did above for catch_reset_n...

val valid_reg = AsyncResetReg(valid, clock, reset, "valid_reg")